config/config.go

```diff
@@ -299,7 +299,7 @@ func (c *Context) DeInitialize(prompter func(...interface{}) bool, returnOnAnyEr
 	}
 
 	for _, p := range pathsToRemove {
-		if !prompter("remove: ", p, ". This operation is permanent (Y/N) ") {
+		if !prompter("remove: ", p, ". This operation is permanent (Y/n) ") {
```

Comment:
How come the change of `Y/n`? I was thinking it is representative of `Yes/No` instead of `Yes/no`.

Reply:
The capitla letter represents the default. https://unix.stackexchange.com/questions/15918/whats-the-standard-used-by-yum-prompt-is-this-ok-y-n